### PR TITLE
Mobile Contact Form Pull Request

### PR DESCRIPTION
**Pull request by MussieTeka**
I have added an updated index.html and style.css to the project to create the contact form part of the website.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -619,3 +619,162 @@ header {
...
+  cursor: pointer;
+}
+
+.gint:hover {
+  opacity: 0.7;
+  color: #fff;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+}
+
```
Comment (Owusu-Desmond) [inline style.css:761]: - [ ] Please make the  `Get in touch`  button follow the button styling: Active, Hover, and Pressed (see design)
  
  ### YOUR DESIGN
  
  image
  
  ### Figma Design
  Screenshot from 2022-12-03 19-12-27

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @MussieTeka and @Otegaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (MussieTeka) [reply style.css]: Thank you for your feedback. I have implemented the suggested changes and added Active, Hover, Pressed, and Disabled button styling to the style.css file as per your remark.

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Button interactions working :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**